util/missingcrypto.txt

```diff
@@ -1562,6 +1562,7 @@ i2s_ASN1_ENUMERATED_TABLE(3)
 i2s_ASN1_IA5STRING(3)
 i2s_ASN1_INTEGER(3)
 i2s_ASN1_OCTET_STRING(3)
+i2s_ASN1_UTF8STRING(3)
```

Comment:
It would be significantly better if you provide the documentation for both functions you've added and the similar old ones. I'm afraid it's impossible to add new undocumented public API now.

Reply:
This functions i similar to i2s_ASN1_IA5STRING than have no documentation. And i think they have no need execept internal use. But if you insist i can add a documentation

Reply:
Yes, and I suggest mentioning all the similar string functions in it.

Reply:
Errr...if its internal then it should not be a public function. (Just because we used to do this in the past is not a reason to do it now).

Reply:
I'm not sure whether it's internal or not. And anyway it's worth documenting it.

Reply:
Documenting it is still desirable whether internal or not. My main concern though is not to add new unnecessary public functions.